lesson07/assignment/wr-vue/src/main.js

```diff
@@ -0,0 +1,5 @@
+import { createApp } from 'vue';
```

Comment:
import { createApp } from 'vue';
import App from './App.vue';
import "bootstrap/dist/css/bootstrap.min.css"
import "bootstrap"
import router from "./router";

createApp(App).use(router).mount('#app')

This is what this file might look like after installing bootstrap (npm i bootstrap), vue-router (npm i vue-router) and hooking up the routing.  When you create a new Vue 3 project, it is probably easiest to have it include the router, and decline everything else including eslint.